Core/include/Acts/Utilities/GridBinFinder.ipp

```diff
@@ -10,10 +11,12 @@ template <std::size_t DIM>
 template <typename... args>
 Acts::GridBinFinder<DIM>::GridBinFinder(args&&... vals) {
   static_assert(sizeof...(args) == DIM);
-  static_assert(std::conjunction<std::disjunction<
-                    std::is_same<int, typename std::decay<args>::type>,
-                    std::is_same<std::vector<std::pair<int, int>>,
-                                 typename std::decay<args>::type>>...>::value);
+  static_assert(
+      std::conjunction<std::disjunction<
+          std::is_same<int, typename std::decay<args>::type>,
+          std::is_same<std::pair<int, int>, typename std::decay<args>::type>,
+          std::is_same<std::vector<std::pair<int, int>>,
+                       typename std::decay<args>::type>>...>::value);
```

Comment:
Are you reimplementing function / constructor overloading here with variadic template parameters? 😅

Reply:
that is just checking the provided inputs are of the allowed types. Do you suggest doing it in a different way?